prospector.yml

```diff
@@ -35,6 +35,15 @@ pep257:
     - D105
     - D211
     - D104
+    - D212  # Multi-line docstring summary should start at the first line
```

Comment:
D213 and D212 are incompatible. We use D213

See: http://pep257.readthedocs.io/en/latest/error_codes.html